src/control/cmd/dmg/system.go

```diff
@@ -286,14 +286,20 @@ func displaySystemAction(log logging.Logger, results system.MemberResults,
 	}
 
 	if absentHosts.Count() > 0 {
```

Comment:
Any reason not to output absentRanks in a similar fashion?

Reply:
absent ranks already get printed in the system action result table (when there are results to print), so if it's already been printed we don't want to print it again as a stand-alone line, I've tidied up the printing logic and added more test coverage in https://github.com/daos-stack/daos/pull/4795